tests/perf/service_invocation_http/service_invocation_http_test.go

```diff
@@ -122,8 +123,37 @@ func TestServiceInvocationHTTPPerformance(t *testing.T) {
 		t.Logf("added latency for %s percentile: %sms", v, fmt.Sprintf("%.2f", latency))
 	}
 
+	err = saveResults(baselineResult, daprResult)
+	if err != nil {
+		t.Error(err)
+	}
+
 	require.Equal(t, 0, daprResult.RetCodes.Num400)
 	require.Equal(t, 0, daprResult.RetCodes.Num500)
 	require.Equal(t, 0, restarts)
 	require.True(t, daprResult.ActualQPS > float64(p.QPS)*0.99)
 }
+
+func saveResults(baseline, dapr perf.TestResult) error {
```

Comment:
Can we add some metadata to it? Like commit id, branch, repo, test name? This can help create reports.

Reply:
We don't get those values in the test run Go code.

Reply:
Actually we do, added.